rules.ini

```diff
@@ -44,7 +44,16 @@ ChromeEngine[] = \.rpack$
 Clausewitz[] = (?:^|/)common/defines\.(?:lua|txt)$
 Clausewitz[] = ^(?:game/)?common/defines/00_defines\.txt$
 ClickTeamFusion = (?:^|/)mmfs2\.dll$
-Cocos2d = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)Cocos2d\.lua$
+Cocos[] = (?:^|/)jsb-adapter/jsb-engine\.jsc?$
+Cocos[] = (?:^|/)jsb-adapter/engine-adapter\.jsc?$
+Cocos[] = (?:^|/)cocos2d-jsb\.[0-9a-z]+\.jsc?$
```

Comment:
you can remove this one, any depot that matches also matches `(?:^|/)jsb-adapter/jsb-engine\.jsc?$`
```suggestion
```

Reply:
Removed since we match `cocos2d` keyword for directory and filename.